$OpenBSD: patch-configure_ac,v 1.11 2016/06/13 11:15:27 ajacoutot Exp $

XXX: Short cut check for shm_open(3).
Remove when/if we have librt.

--- configure.ac.orig	Thu Jun  9 09:09:24 2016
+++ configure.ac	Mon Jun 13 12:38:58 2016
@@ -1796,6 +1796,9 @@ AG_GST_CHECK_FEATURE(SHM, [POSIX shared memory source 
         *-*-OpenBSD*)
             HAVE_SHM=yes
             ;;
+	*-*-openbsd*)
+	    HAVE_SHM=yes
+	    ;;
         *)
             AC_CHECK_LIB([rt], [shm_open],
                 AC_CHECK_DECL([MSG_NOSIGNAL], HAVE_SHM=yes, HAVE_SHM=no, [
